File: NOVEMBER-2025/24-11-2025/1018.cs
public class Solution {
    public IList<bool> PrefixesDivBy5(int[] nums) {
        var ans = new List<bool>();
        int mod = 0;

        foreach (int bit in nums) {
            mod = (mod * 2 + bit) % 5;
            ans.Add(mod == 0);
        }

        return ans;
    }
}

File: NOVEMBER-2025/24-11-2025/1018.dart
class Solution {
  List<bool> prefixesDivBy5(List<int> nums) {
    List<bool> ans = [];
    int mod = 0;

    for (int bit in nums) {
      mod = (mod * 2 + bit) % 5;
      ans.add(mod == 0);
    }

    return ans;
  }
}